docs/monthly-meeting/2022-06.md

```diff
@@ -0,0 +1,175 @@
+# Documentation Community Team Meeting (June 2022)
+
+- **Date:** 2022-06-06
+- **Time:** [21:00 UTC](https://arewemeetingyet.com/UTC/2022-04-04/21:00/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **GitHub issue:** Agenda Planning Issue
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:** Use the "General Voice" channel (it does video); hover over "General Voice" in the side bar and click in the "speech balloon" to bring up the side chat
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+- name / `@github-username`
+- Adam Turner / `@AA-Turner`
+- Guido van Rossum / `@gvanrossum`
+- Petr Viktorin / `@encukou`
+- Pradyun Gedam / `@pradyunsg`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Mariatta / `@mariatta`
+- Ned Batchelder / `@nedbat`
+- Hugo van Kemenade / `@hugovk`
+- Mats Wichmann / `@mwichmann`
+- John P. Rouillard / `@rouilj`
+- Daniele Procida / `@evildmp`
+
+## Quick updates - Introductions
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+- Daniele is the author of Diátaxis.
+
+## Reports and celebrations
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- CPython triage permissions for CAM, Pradyun, Adam, and Ned.
+
+
+## Agenda items
+
+### Past Action Items
+
+- [ ] Carol (and CAM?) will be updating the readme and guide at the sprints.
+
+Update: CAM didn't see any reviews or feedback on his PRs. (He did take some time off from Python work.)
+
+- [ ] Creating a 'docs-approved' (wording to be agreed) label for CPython.
+
+Needs discussion, please open issue on core-workflow repo.
+(There's a template for label changes: https://github.com/python/core-workflow/issues/new/choose)
+
+
+### 'Internal' items
+
+*For and about the Community or Working Group*
+
+#### Docs WG renaming to Editorial Board
+
+> If we rename the WG to an Editorial Board, we should clarify the interpretation of the name.
+>
+> A quarterly meeting for the WG seems like the right cadence.
+> 
+> Editorial Board is the agreed name. Wikipedia has a good definition which aligns with our meaning.
+
+Status: There is currently no ceremony/meetings for the work group/editorial board, to meet and make progress on this.
```

Comment:
Removed "to meet and make progress on this" as I'm not sure anyone suggested the EB having a meeting.
```suggestion
Status: There are currently no meetings for the editorial board.
```